quick-tabs/popup.js

```diff
@@ -770,20 +770,24 @@ AbstractSearch.prototype.executeSearch = function(query, searchBookmark, searchH
   var filteredClosed = [];
   var filteredBookmarks = [];
 
+  function checkTabs(tabs) {
+    return tabs.filter(tab => (tab && tab.title));
+  }
+
   if (query.trim().length === 0) {
     // no need to search if the string is empty
-    filteredTabs = bg.tabs;
+    filteredTabs = checkTabs(bg.tabs);
     filteredClosed = bg.closedTabs;
   } else if (query === "<))") {
-    filteredTabs = bg.tabs.filter(filterAudible)
+    filteredTabs = checkTabs(bg.tabs.filter(filterAudible))
```

Comment:
Same could be done here, rather than filtering the result of a filter, do the check for a `tab.title` within a single filter where `filterAudible` is passed into.

Reply:
Fixed as well.